webapp/src/ts/services/xml-forms.service.ts

```diff
@@ -264,11 +264,14 @@ export class XmlFormsService {
           // fallback for backwards compatibility
           return this.getByView(internalId);
         }
+        console.error('Error in XMLFormService getByView : ', err?.message, err);
         throw err;
       })
```

Comment:
I think we also need to catch errors and log errors from that are thrown by `getByView`, right?
It would be super helpful if you would add unit tests that trigger each of these errors and assert that `console.error` is called with the expected message.

Reply:
yeah, you are right, I tried to add some more, please check it out